doc/api/assert.md

```diff
@@ -257,18 +257,23 @@ property set equal to the value of the `message` parameter. If the `message`
 parameter is undefined, a default error message is assigned.
 
 ## assert.fail(message)
-## assert.fail(actual, expected, message, operator)
+## assert.fail(actual, expected[, message[, operator[, stackStartFunction]]])
```

Comment:
It's not exact. I think it should be:
`assert.fail(actual, expected, message, operator[, stackStartFunction])`